application/src/main/java/org/togetherjava/tjbot/features/github/GitHubUnlinkCommand.java

```diff
@@ -0,0 +1,84 @@
+package org.togetherjava.tjbot.features.github;
+
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.DatabaseException;
+import org.togetherjava.tjbot.db.generated.tables.PrNotifications;
+import org.togetherjava.tjbot.features.CommandVisibility;
+import org.togetherjava.tjbot.features.SlashCommandAdapter;
+
+/**
+ * Slash command to unlink a project from a channel.
+ */
+public class GitHubUnlinkCommand extends SlashCommandAdapter {
+
+    private static final Logger logger = LoggerFactory.getLogger(GitHubUnlinkCommand.class);
+
+    private static final String REPOSITORY_OWNER_OPTION = "owner";
+    private static final String REPOSITORY_NAME_OPTION = "name";
+
+    private final Database database;
+
+    /**
+     * Creates new GitHub unlink command.
+     * 
+     * @param database the database to remove linked pull request notifications
+     */
+    public GitHubUnlinkCommand(Database database) {
+        super("unlink-gh-project", "Unlinks a GitHub repository", CommandVisibility.GUILD);
+        this.database = database;
+
+        getData()
```

Comment:
We don't need the repo name/channel for unlinking or any options. Unless you're designing in a way that multiple repositories can be linked to a single project, in that case you'll also need a `/list-linked-gh-repositories` command